docs/source/guide/tutorials/base/how_to_train/anomaly_detection.rst

```diff
@@ -1,4 +1,190 @@
+
+This is how it should look like in your file system:
+
+.. code-block:: bash
+
+    data/anomaly/hazelnut_toy/
+    ├── colour
+    │   ├── 00.jpg
+    │   ├── 01.jpg
+    │   ...
+    ├── crack
+    │   ├── 01.jpg
+    │   ...
+    ├── good
+    │   ├── 00.jpg
+    │   ├── 01.jpg
+    │   ...
+    ├── LICENSE
+    └── mask
+        ├── colour
+        │   ├── 00.jpg
+        │   ├── 01.jpg
+        │   ...
+        └── crack
+            ├── 01.jpg
+            ├── 02.jpg
+            ...
+
+***************************
+Training
+***************************
+
+For this example let's look at the anomaly detection tasks
+
+.. code-block:: bash
+
+    otx find --task anomaly_detection
+
+::
+
+    +-------------------+-----------------------------+-------+--------------------------------------------------------------+
+    |        TASK       |              ID             |  NAME |                          BASE PATH                           |
+    +-------------------+-----------------------------+-------+--------------------------------------------------------------+
+    | ANOMALY_DETECTION | ote_anomaly_detection_stfpm | STFPM | otx/algorithms/anomaly/configs/detection/stfpm/template.yaml |
+    | ANOMALY_DETECTION | ote_anomaly_detection_padim | PADIM | otx/algorithms/anomaly/configs/detection/padim/template.yaml |
+    +-------------------+-----------------------------+-------+--------------------------------------------------------------+
+
+You can see two anomaly detection models, STFPM and PADIM. For more detail on each model, refer to Anomalib's `STFPM <https://openvinotoolkit.github.io/anomalib/reference_guide/algorithms/stfpm.html>`_ and `PADIM <https://openvinotoolkit.github.io/anomalib/reference_guide/algorithms/padim.html>`_ documentation.
+
+Let's proceed with PADIM for this example. 
+
+.. code-block:: bash
+
+    otx train ote_anomaly_detection_padim --train-data-roots data/anomaly/hazelnut_toy --val-data-roots data/anomaly/hazelnut_toy
+
+This will start training and generate artifacts for commands such as ``export`` and ``optimize``. You will notice the ``otx-workspace-ANOMALY_DETECTION`` directory in your current working directory. This is where all the artifacts are stored.
+
+**************
+Evaluation
+**************
+
+Now that we have trained the model, let's see how it performs on the a specific dataset. In this example we will use the same dataset to generate evaluation metrics. To perform evaluation you need to run the following commands:
+
+.. code-block:: bash
+
+    otx eval ote_anomaly_detection_padim \
+        --test-data-roots data/anomaly/hazelnut_toy \
+        --load-weights otx-workspace-ANOMALY_DETECTION/models/weights.pth \
+        --save-performance otx-workspace-ANOMALY_DETECTION/performance.json
+
+You should see an output similar to the following::
+
+    MultiScorePerformance(score: 0.9032258064516128, primary_metric: None, additional_metrics: (1 metrics), dashboard: (1 metric groups))
```

Comment:
Maybe we could add some explanation about the different score types here (global and local)

Reply:
I have updated it. Let me know if it is fine.